packages/npm/src/utils.ts

```diff
@@ -20,10 +20,9 @@ import * as path from "path";
 import * as exec from "@actions/exec";
 import { IContext, utils } from "@octorelease/core";
 
-export async function npmAddTag(context: IContext, pkgName: string, pkgVersion: string, tag: string, registry: string,
-    inDir?: string): Promise<void> {
-    const registryPrefix = pkgName.startsWith("@") ? `${pkgName.split("/")[0]}:` : "";
-    const cmdArgs = ["dist-tag", "add", `${pkgName}@${pkgVersion}`, tag, `--${registryPrefix}registry=${registry}`];
+export async function npmAddTag(context: IContext, pkgSpec: string, tag: string, registry: string,
```

Comment:
Do any of the other plugins (core ones included) make use of the `npmAddTag` utility ?
If so, would this be considered another breaking change (like the `warning -> warn` from core) ?

Reply:
Even though I still have this question, I approved the PR because I think it's a simple enough change to implement for any plugins 😋

Reply:
Good question, I agree we should be keeping potential breaking changes in mind for the sake of plug-ins 🙂 

In this case, I checked and the Lerna plug-in doesn't directly call the `npmAddTag` method so I think we should be good.